src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const JobsTrainingButtons = ({ setCardsArr, setSearchCards }) => {
+  const { t } = useTranslation();
+  //The selected button is disabled to let the user know which data he is viewing (jobs or training).
+  const [isDisabled, setIsDisabled] = useState(true);
```

Comment:
what really confuses me is that you have a single true/false to represent whether two buttons are disabled. that's the reason why this code is hard to understand. if someone sees "isDisabled", its asking "hey is this button (a single button) disabled?" but somehow this boolean is used to represent whether one of two buttons are disabled.

also it seems to me that one of the buttons is always disabled, is that correct?

Reply:
Yes, one of the buttons is always disabled.  if we are seeing jobs data then jobs button is disabled. if we are seeing training data then training button is disabled.